lib/mayaUsd/utils/utilFileSystem.h

```diff
@@ -49,6 +49,17 @@ std::string getMayaSceneFileDir();
 MAYAUSD_CORE_PUBLIC
 std::string getMayaWorkspaceScenesDir();
 
+/*! \brief takes in an absolute file path and returns the path relative to maya scene file
+ */
+MAYAUSD_CORE_PUBLIC
+std::string getPathRelativeToMayaSceneFile(const std::string& fileName);
+
+/*! \brief returns the flag specifying whether Usd file paths should be saevd as relative to Maya
+ * scene file
```

Comment:
Small typo: saevd -> saved. Not necessary to fix and redo a round-trip of PF + review just for this.